_universum/lib/utils.py

```diff
@@ -135,7 +135,7 @@ def import_module(name, path=None, target_name=None):
 
 def trim_and_convert_to_unicode(line):
     if isinstance(line, str):
```

Comment:
Remove this condition.

Reply:
it will broke logic. I don't want to make refactoring until fix all tests.

Reply:
```
if condition:
    pass
else if another_condition:
    do_something
```
is equal to
```
if another_condition:
    do_something
```